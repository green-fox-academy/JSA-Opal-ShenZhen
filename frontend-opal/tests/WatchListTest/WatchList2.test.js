import React from 'react';
import renderer from 'react-test-renderer';

import WatchList2 from '../../components/WachList2/index';

describe('<WatchList2 />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<WatchList2 />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
