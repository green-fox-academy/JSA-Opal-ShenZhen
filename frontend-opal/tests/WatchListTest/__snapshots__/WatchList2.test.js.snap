// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WatchList2 /> renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "borderColor": "black",
        "borderStyle": "solid",
        "borderWidth": 2,
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <Text>
      My WatchList2
    </Text>
    <Image
      source={1}
      style={
        Object {
          "height": 14,
          "width": 14,
        }
      }
    />
  </View>
</View>
`;
